mlflow/store/file_store.py

```diff
@@ -318,10 +333,14 @@ def _get_run_files(self, run_uuid, resource_type):
         elif resource_type == "tag":
             subfolder_name = FileStore.TAGS_FOLDER_NAME
         else:
-            raise Exception("Looking for unknown resource under run.")
+            raise MlflowException(
```

Comment:
Let's leave the other exception here since it is an unexpected error. (The code should never reach this block.)

Reply:
Good point! Reverted!